darwin/datatypes.py

```diff
@@ -82,3 +82,25 @@ def make_attributes(attributes):
 
 def make_text(text):
     return SubAnnotation("text", text)
+
+
+def make_keyframe(annotation, idx):
```

Comment:
It doesn't look we're using these new functions here, am I missing anything?

Reply:
good catch missed a file.